### PR TITLE
Fix (db/talks) : Defias Traitor now addresses player on say

### PR DESCRIPTION
**Pull request by patou01**
## Changes Proposed:
-  Make last "say" target the stored target instead of self. Using self made him use his last known target which will be an NPC in most cases.
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9146

## Tests Performed:

- 
- 

## How to Test the Changes:

.go c 90214
.q a 142
.q c 142
.q rew 142
take quest from Traitor

Unfortunately it's at the end of the quest so it takes a few minutes. Ideally make sure that the traitor has aggro'd a mob once before finishing and without aggroing anything.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nefertumm — APPROVED** [review]

**Comment by savowe** [comment]
grafik

- [x] tested the escort quest and dialogue

File: data/sql/updates/pending_db_world/rev_1636804117557511400.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1636804117557511400');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 467;
+
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1636804117557511400.sql:4]: this looks to me like a useless line

Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1636804117557511400.sql:4]: @patou01

Comment (patou01) [reply data/sql/updates/pending_db_world/rev_1636804117557511400.sql:4]: I mean yeah, I just take whatever keira gives me... 
image

We can remove it, it should already have smartAI.

**Pull request merged**